docs/tutorials/join-dal-baker/verify-rights.md

```diff
@@ -61,6 +61,39 @@ Follow these steps to verify that your DAL node is receiving attestation rights:
 
          If this command returns a list of future attestation rights for your account, the delay has not expired yet and you must wait for that cycle to arrive.
 
+   - Check to see if you are active and re-register as a delegate if necessary:
+
+      1. Run this command to see if your account is active:
+
+         ```bash
+         octez-client rpc get /chains/main/blocks/head/context/delegates/$MY_BAKER | jq . | grep deactivated
+         ```
+
+         Baker accounts are deactivated when the baker is offline for a certain time.
```

Comment:
Bakers are deactivated if the baker has not participated at all in consensus before a  given ` grace_period`   has elapsed, denoted by a cycle number.  This can be consulted with the following RPC endpoint,

```
octez-client rpc get /chains/main/blocks/head/context/delegates/baker/grace_period
```

Reply:
The grace period semantics is a bit tricky. I'm not sure we need it here. But we should point to a relevant article in the octez doc (cc @NicVolanschi )

Reply:
Removed the stuff about the grace period.